php/ext/google/protobuf/protobuf.c

```diff
@@ -225,8 +225,8 @@ EnumDescriptorInternal* get_proto_enumdesc(const char* proto) {
 #ifndef NDEBUG
```

Comment:
Can we just remove this entire function? It does not seem to be called anywhere.